src/main/java/bio/terra/Main.java

```diff
@@ -26,7 +26,7 @@ public static void main(String[] args) throws Exception {
         SpringApplication theApp = new SpringApplication(Main.class);
         // Initially, Jade runs only with bigquery, so we set the profile here.
```

Comment:
nit: "only runs with bigquery and cloud storage, so we set the google profile here"